data/morphology/grab_unimorph_data.py

```diff
@@ -1,52 +1,69 @@
 #!/usr/bin/env python
 """Downloads UniMorph morphological paradigms data."""
-
 import json
 import logging
 import os
-import requests
 import time
-
 from typing import Dict
 
 
+""" import requests in separate code block because of third party status """
+import requests
+
+
+start_time = time.time()
+
+
 UNIMORPH_DICT_PATH = "unimorph_languages.json"
 
 
-def download(data_to_grab: Dict[str, str]) -> Dict[str, str]:
-    to_retry = {}
-    os.mkdir("tsv")
-    for language, url in data_to_grab.items():
-        with requests.get(url, stream=True) as response:
-            logging.info("Downloading: %s", language)
+def download():
+    os.makedirs("unimorph_for_split", exist_ok=True)
```

Comment:
I prefered the old line 18.

1. The name `tsv` is short and sweet and compatible with names used elsewhere, plus they're TSV files, plus the fact that they're for splitting doesn't need to be encoded here.
2. I don't like `exist_ok` here because somebody could have a mix of old and new files and not know it. If they want to remove the old directory, make them do so.

 (Having it overwrite the old directory partially is dangerous and tends to be buggy. If someone wants to remove teh old directory, they can.)